Make 'nix store gc' use the auto-GC policy (#7851)
NixOS/nix: src/nix/store-gc.md

  # nix store gc
  ```

* Perform garbage collection if there is less than 1 GiB of free space

>   in `/nix/store`, and stop once there is at least 5 GiB of free

Reviewer: ```suggestion
  on the file system partition holding `/nix/store`, and stop once there is at least 5 GiB of free
```

A directory doesn't have free space, right?

Reviewer 2: Technically a file system can span many partitions (RAID, etc.) so I would just say "file system" and drop "partition".

Reviewer: Right. Edited.

Author: > A directory doesn't have free space, right?

Well, a directory is always on a particular file system, so it can be said to have free space.

Reviewer 2: ```suggestion
  for `/nix/store`, and stop once there is at least 5 GiB of free
```
here is a lighter-weight alternative: "for" as in "for `/nix/store` to use". Exactly who or what is providing the free space, or how it is being provided, is left unstated as before.

  space.

  ```console

>   # nix store gc --gc-threshold 1G --gc-limit 5G

Reviewer 3: ```suggestion
  # nix store gc --gc-threshold 10G --gc-limit 30G
```

These numbers are more realistic, allowing for larger builds without consuming much space.

  ```

# Description

This command deletes unreachable paths in the Nix store, observing the

> GC policy configured by the

Reviewer: ```suggestion
garbage collection policy configured by the
```

> [`gc-threshold`](../conf-file.md#conf-gc-threshold) and
> [`gc-limit`](../conf-file.md#conf-gc-limit) configuration

Reviewer: ```suggestion
[`gc-threshold`](@docroot@/command-ref/conf-file.md#conf-gc-threshold) and
[`gc-limit`](@docroot@/command-ref/conf-file.md#conf-gc-limit) configuration
```
